cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -137,7 +138,10 @@ public void testCreateWithConfig() throws Exception {
     // prepare - create namespace with config in its properties
     String propertiesString = String.format("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":%s}",
```

Comment:
Is there a better way to get this string? Escaping looks ugly. We might want to add more properties in the future.